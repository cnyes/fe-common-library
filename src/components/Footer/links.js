const footerLinks = {
  navs: [
    {
      title: '關於我們',
      name: 'about',
      url: 'http://www.cnyes.com/cnyes_about/cnyes_about.html',
    },
    {
      title: '廣告服務',
      name: 'ad',
      url: 'http://www.cnyes.com/cnyes_about/cnyes_AD01.html',
    },
    {
      title: '金融資訊元件',
      name: 'financial',
      url: 'http://www.cnyes.com/cnyes_about/cnyes_pas01.html',
    },
    {
      title: '聯絡我們',
      name: 'contact',
      url: 'http://www.cnyes.com/cnyes_about/cnyes_ctcUsTpe.html',
    },
    {
      title: '徵才',
      name: 'job',
      url: 'http://www.104.com.tw/jobbank/custjob/index.php?r=cust&j=5e6042253446402330683b1d1d1d1d5f2443a363189j01',
    },
    {
      title: '網站地圖',
      name: 'sitemap',
      url: 'http://www.cnyes.com/cnyes_about/site_map.html',
    },
    {
      title: '法律聲明',
      name: 'legal',
      url: 'http://www.cnyes.com/cnyes_about/cnyes_sos01.html',
    },
  ],
  socials: [
    {
      title: '粉 絲 團',
      name: 'facebook',
      url: 'https://www.facebook.com/cnYES/',
    },
    {
      title: '鉅亨網Line',
      name: 'line',
      url: 'https://line.me/ti/p/@ZLU0489G',
    },
    {
      title: '鉅亨網APP',
      name: 'app',
      url: 'http://www.cnyes.com/app?utm_source=cnyes&utm_medium=desktop&utm_campaign=desktop_footer',
    },
  ],
};

export default footerLinks;
