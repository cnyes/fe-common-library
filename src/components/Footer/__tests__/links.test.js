/* eslint-disable import/first,func-names,global-require,prefer-arrow-callback */
import links from '../links';

describe('links', () => {
  it('should make a snapshot', () => {
    expect(links).toMatchSnapshot();
  });
});
