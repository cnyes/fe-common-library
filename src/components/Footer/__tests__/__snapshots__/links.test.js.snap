// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`links should make a snapshot 1`] = `
Object {
  "navs": Array [
    Object {
      "name": "about",
      "title": "關於我們",
      "url": "http://www.cnyes.com/cnyes_about/cnyes_about.html",
    },
    Object {
      "name": "ad",
      "title": "廣告服務",
      "url": "http://www.cnyes.com/cnyes_about/cnyes_AD01.html",
    },
    Object {
      "name": "financial",
      "title": "金融資訊元件",
      "url": "http://www.cnyes.com/cnyes_about/cnyes_pas01.html",
    },
    Object {
      "name": "contact",
      "title": "聯絡我們",
      "url": "http://www.cnyes.com/cnyes_about/cnyes_ctcUsTpe.html",
    },
    Object {
      "name": "job",
      "title": "徵才",
      "url": "http://www.104.com.tw/jobbank/custjob/index.php?r=cust&j=5e6042253446402330683b1d1d1d1d5f2443a363189j01",
    },
    Object {
      "name": "sitemap",
      "title": "網站地圖",
      "url": "http://www.cnyes.com/cnyes_about/site_map.html",
    },
    Object {
      "name": "legal",
      "title": "法律聲明",
      "url": "http://www.cnyes.com/cnyes_about/cnyes_sos01.html",
    },
  ],
  "socials": Array [
    Object {
      "name": "facebook",
      "title": "粉 絲 團",
      "url": "https://www.facebook.com/cnYES/",
    },
    Object {
      "name": "line",
      "title": "鉅亨網Line",
      "url": "https://line.me/ti/p/@ZLU0489G",
    },
    Object {
      "name": "app",
      "title": "鉅亨網APP",
      "url": "http://www.cnyes.com/app?utm_source=cnyes&utm_medium=desktop&utm_campaign=desktop_footer",
    },
  ],
}
`;
